__tests__/parse.spec.js

```diff
@@ -375,6 +375,30 @@ Index: text.txt
     expect(file.chunks[0].changes.length).toBe(1);
   });
 
+  it("should parse GitHub API patch diff when listing files of a pull request", function () {
+    const diff = `@@ -1 +1 @@
+-hello world
++hello universe`;
+
+    const files = parse(diff);
+    expect(files.length).toBe(1);
+    const [file] = files;
+    expect(file.chunks[0].content).toBe(
+      "@@ -1 +1 @@"
+    );
+    expect(file.from).toBeUndefined();
```

Comment:
@mikedidomizio we can also pass `from` and `to` filenames as options to `parse` function. WDYT?

Reply:
Hey @sergeyt thanks for looking at my pull request.  I thought about what you said and passing the `from` and `to` to the `parse` function and considered it, but I think we shouldn't.

1.  The `parse` function should strictly parse the information we pass into it.  In this situation the data doesn't have any that data, it should tell us that nothing is in there.  It's accurate about the information we provided.
2. If we do pass the information to the parse function to return it, and someday it does send us the real data, we might be overriding the values (if we don't put that into consideration).

I think we should leave the `parse` function the way it is.